dev_docs/source/conf.py

```diff
@@ -9,23 +9,39 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-#
+
 import os
 import sys
+from pathlib import Path
 
-sys.path.insert(0, os.path.abspath("."))
-sys.path.insert(0, os.path.abspath(".."))
-sys.path.insert(0, os.path.abspath("../.."))
+ROOT_PATH = Path(__file__).resolve().parents[2]
+DEV_DOCS_PATH = ROOT_PATH / "dev_docs"
+SRC_PATH = ROOT_PATH / "flood_adapt"
+MODULE_PATHS = [SRC_PATH / module for module in ["integrator", "object_model", "api"]]
 
+for path in [ROOT_PATH, DEV_DOCS_PATH, SRC_PATH, *MODULE_PATHS]:
+    assert path.exists(), f"Path does not exist: {path}"
+    sys.path.insert(0, os.path.abspath(path))
 
 # -- Project information -----------------------------------------------------
 
 project = "FloodAdapt-API"
 copyright = "2024, Adrichem"
 author = "Adrichem"
 
+
 # The full version, including alpha/beta/rc tags
-release = "0.0.1"
+def get_floodadapt_version():
+    version = None
+    with open(ROOT_PATH / "flood_adapt" / "__init__.py", "r") as f:
+        for line in f:
+            if line.startswith("__version__"):
+                version = line.split("=")[1].strip().strip('"')
+    assert version, "Could not find version in __init__.py"
+    return version
+
+
+release = get_floodadapt_version()
```

Comment:
Nitpicky and up to you, but why not `from flood_adapt import __version___`, seems cleaner imo. Or is flood_adapt not installed in the docs environment?

Reply:
whoops, this is part of an old solution and not actually used anymore. so I'll remove it